src/diff/changes/apiChanges.ts

```diff
@@ -6,6 +6,7 @@
  */
 import { NodeChanges } from "./nodeChanges";
 import { RuleCategory } from "../ruleSet";
+import _ from "lodash";
```

Comment:
Avoid using lodash as we are planning to get rid of it?

Reply:
Aww, I was hoping no one would notice so I could be lazy.

Reply:
I agree with this sentiment if you are using new lodash functionality.  If this is using lodash methods that we already use elsewhere in this project then don't try to fix it here,